bundles/edu.kit.kastel.sdq.eclipse.grading.view/src/edu/kit/kastel/eclipse/grading/view/assessment/CustomButtonDialog.java

```diff
@@ -104,14 +105,31 @@ protected Control createDialogArea(Composite parent) {
 																	// determined internally.
 			final Label customPenaltyLabel = new Label(comp, SWT.RIGHT);
 			customPenaltyLabel.setText("Custom Penalty: ");
```

Comment:
It would probably better to change the name here to make it clear that we now can give bonus points

Reply:
```suggestion
			customPenaltyLabel.setText("Custom Points: ");
```

Would that work @majuwa